http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -965,6 +974,62 @@ class GKEInterpreter[F[_]](
 
     } yield ()
 
+  private[util] def installCromwellApp(helmAuthContext: AuthContext,
+                                       appName: AppName,
+                                       release: Release,
+                                       cluster: KubernetesCluster,
+                                       nodepoolName: NodepoolName,
+                                       namespaceName: NamespaceName)(implicit ev: Ask[F, AppContext]): F[Unit] = {
+    val chart = config.cromwellAppConfig.chart
```

Comment:
Maybe add a comment to use the `chart` from the DB instead of config when BW-860 is fixed.